src/math/compare.php

```diff
@@ -51,3 +51,15 @@ function minva<T as num>(
   }
   return $min;
 }
+
+/**
+ * Returns wheter a num is NAN.
```

Comment:
Looks good, but `whether`

Reply:
also the comment doesn't add anything to the function name.

Probably say "the not-a-number special float value" or similar

Reply:
I did not see this comment earlier. Sorry for the long delay.